tests/test_series.py

```diff
@@ -126,7 +126,7 @@ def test_series_meta():
         flimage.FLImage(h5file=h5file,
                         meta_data={"pixel size": 0.054e-6},
                         h5mode="r")
-    except (OSError, RuntimeError):
+    except (OSError, RuntimeError, KeyError):
```

Comment:
It seems like later versions of h5py raise a KeyError instead. Exception message "no write intent on file" is the same, though.